_episodes/07-aliases.md

```diff
@@ -77,3 +60,63 @@ $ git hi
 - How many times should you wait before aliasing a command?
 - Do you believe a list of generic two-letter acronyms for common commands will
   save your time?
+
+---
+
+## List of aliases the instructors use
+
+You are welcome to reuse, suggest, improve.
+
+Common aliases:
+
+```
+$ git config --global alias.br branch
+$ git config --global alias.ci "commit -v"
+$ git config --global alias.co checkout
+$ git config --global alias.di diff
+$ git config --global alias.di diff --cached --color-words
+$ git config --global alias.diw "diff --color-words"
+$ git config --global alias.dis "!git --no-pager diff --stat"
+$ git config --global alias.fe fetch
+$ git config --global alias.graph "log --all --graph --decorate --oneline"
+$ git config --global alias.rem remote
+$ git config --global alias.st status
+```
+
+Some common `-p` aliases:
+
+```
+$ git config --global alias.ap "add --patch"
+$ git config --global alias.cip "commit --patch -v"
+$ git config --global alias.cop "checkout --patch"
+```
+
+Working with Git submodules (nested Git repositories):
+
+```
+$ git config --global alias.cl "clone --recursive"
+$ git config --global alias.su "submodule update --init --recursive"
+```
+
+Other aliases that may be too advanced, but would set people up for the future better (not formatted properly):
```

Comment:
Are these too advanced?

I guess I would leave in a separate block at bottom with big disclaimer than they are advanced but you might want to know what is possible.

Reply:
I will rephrase and also remove the "not formatted properly" - forgot that.